docs/examples/LeftTabs.js

```diff
@@ -1,9 +1,24 @@
 const tabsInstance = (
-  <Tabs defaultActiveKey={2} position="left" tabWidth={3}>
-    <Tab eventKey={1} title="Tab 1">Tab 1 content</Tab>
-    <Tab eventKey={2} title="Tab 2">Tab 2 content</Tab>
-    <Tab eventKey={3} title="Tab 3" disabled>Tab 3 content</Tab>
-  </Tabs>
+  <TabContainer id="left-tabs-example" defaultActiveKey="first">
```

Comment:
I can, I didn't know if maybe that was to much noise over the API. I guess the using the plain classes is also a bit confusing